packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.types.ts

```diff
@@ -6,7 +6,12 @@ import { DirectionalHint } from '../../common/DirectionalHint';
 import { IRefObject, IStyleFunctionOrObject } from '../../Utilities';
 import { IStyle, ITheme } from '../../Styling';
 
-export interface ITooltipHost {}
+export interface ITooltipHost {
+  /**
+   * Hides the tooltip
+   */
+  hideTooltip: () => void;
```

Comment:
how about just 'dismiss'. It's a bit redundant to get a reference to the tooltip, and tell it 'hideTooltip'.